internal/docs/customMetricEquivalents.md

```diff
@@ -0,0 +1,17 @@
+| **Container Insights Metric**                       | **Prometheus Equivalent**                                                                                             |
+|-----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **cpuUsageMillicores**                              | `container_cpu_usage_seconds_total`                                                                                                                         |
```

Comment:
kaveesh - please change cpuusagemillicores to rate . and podcount to uniquily group by name,namespace,cluster.
Also think all metrics should filter/group by cluster label, as amw can have many clusters ?